src/Paket.Core/PublicAPI.fs

```diff
@@ -53,27 +55,38 @@ type Dependencies(dependenciesFileName: string) =
         else
             DependenciesFile(dependenciesFileName, InstallOptions.Default, [], [], []).Save()
         Dependencies(dependenciesFileName)
-        
+    
     /// Converts the solution from NuGet to Paket.
-    static member ConvertFromNuget(force: bool,installAfter: bool,initAutoRestore: bool,credsMigrationMode: string option) : unit =        
-        let credsMigrationMode = credsMigrationMode |> Option.map NuGetConvert.CredsMigrationMode.Parse
-        let dependencies = 
-            try Some <| Dependencies.Locate()
-            with _ -> None 
-        
-        let existingDependenciesFile = 
-            if force 
-            then dependencies |> Option.map (fun d -> d.DependenciesFile)
-            else dependencies |> Option.map (fun d -> failwithf "%s already exists, use --force to overwrite" d.DependenciesFile)
-        
-        let dependenciesFileName = 
-            match existingDependenciesFile with
-            | Some file -> file
-            | None -> Path.Combine(Environment.CurrentDirectory, Constants.DependenciesFileName)
+    static member ConvertFromNuget(force: bool,installAfter: bool,initAutoRestore: bool,credsMigrationMode: string option) : unit =
+        let currentDirectory = DirectoryInfo(Environment.CurrentDirectory)
+        let rootDirectory = defaultArg (PaketEnv.locatePaketRootDirectory(currentDirectory)) currentDirectory
 
         Utils.RunInLockedAccessMode(
-            Path.GetDirectoryName(dependenciesFileName),
-            fun () ->  NuGetConvert.ConvertFromNuget(dependenciesFileName, force, installAfter, initAutoRestore, credsMigrationMode))
+            rootDirectory.FullName,
+            fun () ->
+                NuGetConvert.convertR rootDirectory force credsMigrationMode
+                |> either 
+                    (NuGetConvert.replaceNugetWithPaket initAutoRestore installAfter) 
+                    (List.iter (string >> Logging.traceError))
+        )
+
+     /// Converts the current package dependency graph to the simplest dependency graph.
+    static member Simplify(): unit = Dependencies.Simplify(false)
+
+    /// Converts the current package dependency graph to the simplest dependency graph.
+    static member Simplify(interactive : bool) =
+        match PaketEnv.locatePaketRootDirectory(DirectoryInfo(Environment.CurrentDirectory)) with
+        | Some rootDirectory ->
+            Utils.RunInLockedAccessMode(
+                rootDirectory.FullName,
+                fun () -> 
+                    PaketEnv.fromRootDirectory rootDirectory
+                    >>= Simplifier.ensureNotInStrictMode
+                    >>= Simplifier.simplify interactive
+                    |> either (Simplifier.updateEnvironment) (List.iter (string >> Logging.traceError))
```

Comment:
OK, then for now we can do

    |> returnOrFail |> Simplifier.updateEnvironment

instead of 

    |> either (Simplifier.updateEnvironment) (List.iter (string >> Logging.traceError))